float64 x_error
float64 y_error
float64 theta_error